src/main/java/Action.java

```diff
@@ -0,0 +1,150 @@
+import java.lang.reflect.Array;
+import java.util.*;
+
+public class Action {
+    private String commandName;
+    private Item item;
+    private Room room;
+
+    public Action(String commandName, Item item, Room room) {
+        this.commandName = commandName;
+        this.item = item;
+        this.room = room;
+    }
+
+    public static String printInventory(ArrayList<Item> playerItems) {
+        String playerInventory = "";
+        String playerInventoryFinal;
+
+        for (Item item : playerItems){
+            playerInventory = item.getNameFromItem() + ", " + playerInventory;
+        }
+        int unnecessaryComma = playerInventory.lastIndexOf(",");
+        playerInventoryFinal = "Your current inventory: " + playerInventory.substring(0, unnecessaryComma);
+
+        return playerInventoryFinal;
+    }
+
+    public static void doAction(String[] command, Player player) {
+        Action result;
+        String keyword = command[0];
+
+        switch (keyword) {
+            case "move":
+                if (command[1].equals("to")) {
+                    Room currentRoom = player.getCurrentRoom();
+                    String wholeName;
```

Comment:
maybe cleaner imo:

wholeName = command[2];
if (command.length > 3) {
   wholename += " " + command[3]; (dont know if this line is how Java works, but otherwise append or whatever)
}